### ISSUE TITLE
Hello, I have a question about si-sdr.py at your source_code.

### ISSUE THREAD

**Issue by brotherhoon-ml** [wangkenpu/Conv-TasNet-PyTorch#6]
I am referring to your loss function during the implementation of Conv-tasnet.
In the paper, si-snr is used as loss function, but in your code, si-sdr is used.
I think the code is the same as si-snr, can you tell me why you named it si-sdr?

**Comment by brotherhoon-ml** [comment]
haha~ it is same meaning
si-sdr == si-snr

**Comment by wangkenpu** [comment]
Yes. Some researchers may call it SI-SNR, while someone may call it SI-SDR. It actually is the same thing.
